CDC_Backend/templates/verify_email.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="x-apple-disable-message-reformatting">
+    <title></title>
+    <!--[if mso]>
+    <noscript>
+        <xml>
+            <o:OfficeDocumentSettings>
+                <o:PixelsPerInch>96</o:PixelsPerInch>
+            </o:OfficeDocumentSettings>
+        </xml>
+    </noscript>
+    <![endif]-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="shortcut icon" href="favicon.ico"/>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <style>
+        table, td, div, h1, p {
+            font-family: 'Roboto', sans-serif;
+        }
+
+    </style>
+</head>
+<body style="margin:0;padding:0;">
+<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;background:#ffffff;">
+    <tr>
+        <td align="center" style="padding:0;">
+            <table role="presentation"
+                   style="width:602px;border-collapse:collapse;border:1px solid #334878;border-spacing:0;text-align:left;">
+                <tr>
+                    <td align="center" style="padding:40px 0 30px 0;background:#334878;">
+                        <img src="./image.png" alt="" width="200" style="height:auto;display:block;"/>
+                    </td>
+                </tr>
+                <tr>
+                    <td style="padding:36px 30px 42px 30px;">
+                        <table role="presentation"
+                               style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+                            <tr>
+                                <td style="padding:0 0 36px 0;color:#153643;">
+                                    <h1 style="font-size:24px;margin:0 0 20px 0;font-family: 'Roboto', sans-serif;">Hi,</h1>
+                                    <p style="margin:0 0 12px 0;font-size:16px;line-height:24px;font-family: 'Roboto', sans-serif;">
+                                        You have successfully filled the job notification form for <b>{{ job_designation }}</b> at <b>{{ company_name }}</b>. Kindly verify your email by clicking <a href={{ one_time_link }}>here</a>.
```

Comment:
Change the Message to this
`We have received your Job Notification for {{designation}}. Kindly verify your email by clicking <a href={{ one_time_link }}>here</a>`

Reply:
Resolved in commit a9b128d